src/java/fr/paris/lutece/plugins/forms/web/admin/FormStepJspBean.java

```diff
@@ -295,6 +294,13 @@ public String doCreateStep( HttpServletRequest request ) throws AccessDeniedExce
         checkUserPermission( Form.RESOURCE_TYPE, String.valueOf( _step.getIdForm( ) ), FormsResourceIdService.PERMISSION_MODIFY, request, ACTION_CREATE_STEP );
         checkWorkgroupPermission(_step.getIdForm( ), request);
 
+        Step finalStep = StepHome.getFinalStep( _step.getIdForm( ) );
+        if ( finalStep != null )
+        {
+            finalStep.setFinal( false );
```

Comment:
how could we know that the step we are creating is the last one ? it could be a step inserted between two others ?